### PR TITLE
Create plant_genomics_assembly

### PR DESCRIPTION
**Pull request by DanFaria**

### CODE REVIEW

File: pages/tool_assembly/plant_genomics_assembly [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## How can you access the plant genomics tool assembly?
+All the components of this tool assembly are publicly available, but most require registration. So anyone can access the tool assembly provided they register for each tool that requires it.
+
+
+## For what purpose can you use the plant genomics tool assembly?
+<!--- TODO: Add figure --->
+
+### Metadata collection and tracking
+Accurate description of the plant biological materials and samples is critical for interoperability, and should comply with the [MIAPPE](https://www.miappe.org/) standard.
```
Comment (korbinib) [inline pages/tool_assembly/plant_genomics_assembly:23]:
```suggestion
Accurate [documentation](metadata_management.html) of the plant biological materials and samples is critical for interoperability, and should comply with the [MIAPPE](https://www.miappe.org/) standard.
```

File: pages/tool_assembly/plant_genomics_assembly [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+This information should be submitted to [BioSamples](https://www.ebi.ac.uk/biosamples/), with MIAPPE compliance validated using BioSamples' [plant-miappe.json](https://github.com/EBIBioSamples/biosamples-v4/blob/biohackathon_miappe_checklist/webapps/core/src/main/resources/schemas/certification/plant-miappe.json) template.
+Submission of sample descriptions to BioSamples can be done as early as the data collection stage, but at the latest, must acompany submission of the genomic data to the [European Nucleotide Archive](https://www.ebi.ac.uk/ena/browser/home) (ENA) or of genotyping data to the [European Variation Archive](https://www.ebi.ac.uk/eva/) (EVA).
+[e!DAL-PGP](https://edal-pgp.ipk-gatersleben.de/) can be used to manage and share experimental metadata, as well as data.
+
+### Data processing and analysis
+Reference genomes for genome assembly and annotation should be obtained from [ENSEMBL Plants](https://plants.ensembl.org/index.html), if available.
```
Comment (korbinib) [inline pages/tool_assembly/plant_genomics_assembly:29]: Could this go to a reuse section?

File: pages/tool_assembly/plant_genomics_assembly [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Genetic variant data must be produced in the VCF format, and validated using the EVA vcf-validator (https://github.com/EBIvariation/vcf-validator).
+
+### Data sharing and publishing
+Plant genomic data should be submitted to ENA, whereas plant genotyping data should be submitted to EVA.
```
Comment (korbinib) [inline pages/tool_assembly/plant_genomics_assembly:33]:
```suggestion
Plant genomic data should be submitted to ENA together with metadata compliant to the [GSC MIxS plant associated checklist](https://www.ebi.ac.uk/ena/browser/view/ERC000020),  whereas plant genotyping data should be submitted to EVA.
```

File: pages/tool_assembly/plant_genomics_assembly [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+Additionally, data can also be published in e!DAL-PGP.
+
+## Tools used within the tool assembly?
+
+{% include toollist.html tag="<!---plant_geno_assembly--->" %}
```
Comment (korbinib) [inline pages/tool_assembly/plant_genomics_assembly:38]:
```suggestion
{% include toollist.html tag="plant_geno_assembly" %}
```

**Review by korbinib — CHANGES_REQUESTED** [review]

**Review by korbinib — CHANGES_REQUESTED** [review]
Thank you @DanFaria! I have added a few suggestions to the stub.

**Review by korbinib — APPROVED** [review]

File: pages/tool_assembly/plant_genomics_assembly (hunk 2 above)
Comment (DanFaria) [reply pages/tool_assembly/plant_genomics_assembly:29]: Do you mean the ENSEMBL Plants part? I'm not sure.
I tend to see the reuse section as ensuring the reuse of the data in my study, as opposed to reusing data from others in my study.

**Pull request merged**